### PR TITLE
fix(montoring): ensure activation record is always persisted

### LINKED ISSUE #63: [monitoring] Add X-OW-EXTRA-LOGGING: on
If we decide we do https://github.com/adobe/helix-publish/issues/403, we will need this.

### PR DESCRIPTION
**Pull request by rofe**
Fix #63

### CODE REVIEW

File: src/newrelic/monitor_script.js [hunk 1]
```diff
@@ -16,67 +16,73 @@
...
-    if (status.error) {
-      $util.insights.set('errorStatus', status.error.statuscode);
-      $util.insights.set('errorURL', status.error.url);
+$http.get({
+  url: '$$$URL$$$',
+  headers: {
+    // make sure activation records gets persisted
+    'X-OW-EXTRA-LOGGING': 'on',
+  },
+},
```
Comment (rofe) [inline src/newrelic/monitor_script.js:25]: This is the actual change, the rest of the file is just whitespace adjustments

Comment (stefan-guggisberg) [reply src/newrelic/monitor_script.js:25]: why change indention?

File: src/newrelic/monitor_script.js [hunk 2]
```diff
@@ -16,67 +16,73 @@
...
-    ['x-openwhisk-activation-id', 'x-request-id', 'x-version'].forEach((h) => {
-      $util.insights.set(h, response.headers[h]);
-    });
-    if (status.error) {
-      $util.insights.set('errorStatus', status.error.statuscode);
-      $util.insights.set('errorURL', status.error.url);
+$http.get({
+  url: '$$$URL$$$',
+  headers: {
+    // make sure activation records gets persisted
```
Comment (stefan-guggisberg) [inline src/newrelic/monitor_script.js:22]:
```
// make sure activation record gets persisted
```

File: src/newrelic/monitor_script.js (hunk 1 above)
Comment (rofe) [reply src/newrelic/monitor_script.js:25]: Because eslint didn't like it anymore after I turned the changed argument from string to an object 😄

**Pull request merged**